module/Entra/AdditionalFunctions/Remove-EntraPolicy.ps1

```diff
@@ -0,0 +1,81 @@
+function Remove-EntraPolicy {
+    [CmdletBinding(DefaultParameterSetName = '')]
+    param (
+    [Parameter(Mandatory = $true, ValueFromPipeline = $true, ValueFromPipelineByPropertyName = $true)]
+    [System.String] $Id
+    )
+
+    PROCESS {    
+        $params = @{}
+        $customHeaders = New-EntraCustomHeaders -Command $MyInvocation.MyCommand
+        $array = ("activityBasedTimeoutPolicies",	"defaultAppManagementPolicy",	"appManagementPolicies",	"authenticationFlowsPolicy",	"authenticationMethodsPolicy",	"claimsMappingPolicies",	"featureRolloutPolicies",	"homeRealmDiscoveryPolicies",	"permissionGrantPolicies",	"tokenIssuancePolicies",	"tokenLifetimePolicies")
+    
+        foreach ($a in $array) {
```

Comment:
If you give an array a more meaningful name, this would be `foreach ($policyType in $policyTypes) {`.

Reply:
fixed